### DISCUSSION TITLE
Adonis 4.1 websockets

### DISCUSSION THREAD

**Discussion by steveyout** [adonisjs/core#2129] [Help]
How can i intergrate socket.io with adonis 4.1 and also use the ws controllers

**Comment by ehsanquddusi** [comment]
Create `start/socket.js` file and paste following code inside it.

```js
const Server = use('Server')
const io = use('socket.io')(Server.getInstance())

io.on('connection', function (socket) {
  console.log(socket.id)
})
```
For discussion visit [Integrating socket.io with Adonis 4](https://forum.adonisjs.com/t/integrating-socket-io-with-adonis-4/519)

**Reply by steveyout** [reply]
What about in ws controllers

**Reply by davebot9000** [reply]
It is worth noting that the forum links will not work, such as the one above. This is due to the forum being shut down.
